src/config/models.rs

```diff
@@ -29,6 +29,17 @@ pub struct ConfigOptsBuild {
     /// The public URL from which assets are to be served [default: /]
     #[clap(long, parse(from_str=parse_public_url))]
     pub public_url: Option<String>,
+    /// Build without default features [default: false]
+    #[clap(long)]
+    #[serde(default)]
+    pub no_default_features: bool,
+    /// Build with all features [default: false]
+    #[clap(long)]
+    #[serde(default)]
+    pub all_features: bool,
+    /// The public URL from which assets are to be served [default: /]
```

Comment:
Looks like the docstring needs to be updated here.

Reply:
This is the new docstring:
```rust
/// A comma-separated list of features to activate, must not be used with all_features [default: ""]
#[clap(long)]
pub features: Option<String>,
```

The new output for `trunk build -h`:
```
trunk-build 
Build the Rust WASM app and all of its assets

USAGE:
    trunk build [OPTIONS] [TARGET]

ARGS:
    <TARGET>    The index HTML file to drive the bundling process [default: index.html]

OPTIONS:
        --all-features               Build with all features [default: false]
    -d, --dist <DIST>                The output dir for all final assets [default: dist]
        --features <FEATURES>        A comma-separated list of features to activate, must not be
                                     used with all-features [default: ""]
        --filehash <FILEHASH>        Whether to include hash values in the output file names
                                     [default: true]
    -h, --help                       Print help information
        --no-default-features        Build without default features [default: false]
        --public-url <PUBLIC_URL>    The public URL from which assets are to be served [default: /]
        --release                    Build in release mode [default: false]
```

Let me know if this works for you/the team;
I'll get to writing the changelog & readme stuff in the meantime.